src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py

```diff
@@ -272,6 +334,15 @@ def act(self, observations: Observations) -> Dict[str, int]:
         )
 
 
+def convert_to_robot_action(cont_action):
```

Comment:
what does do? can we add some more documentation?

Reply:
converts policy output habitat actions to home-robot actions. Added.